templates/travis/.travis/install.sh.j2

```diff
@@ -55,7 +58,20 @@ else
   PULP_CERTGUARD=git+https://github.com/pulp/pulp-certguard.git
 fi
 
-cat > vars/vars.yaml << VARSYAML
+if [ -n "$TRAVIS_TAG" ]; then
+  # Install the plugin only and use published PyPI packages for the rest
+  cat > vars/vars.yaml << VARSYAML
+---
+images:
+  - ${PLUGIN}-${TAG}:
+      image_name: $PLUGIN
+      tag: $TAG
+      plugins:
+        - pulp-certguard
```

Comment:
I agree with this design, but
1. In general, we need to do a pulp-certguard release at the time we release pulpcore & pulpcore-plugin.